lib/foreman_remote_execution_core.rb

```diff
@@ -10,12 +10,37 @@ module ForemanRemoteExecutionCore
                     :kerberos_auth           => false,
                     :async_ssh               => false,
                     # When set to nil, makes REX use the runner's default interval
-                    :runner_refresh_interval => nil)
+                    :runner_refresh_interval => nil,
+                    :ssh_log_level           => :fatal)
+
+  SSH_LOG_LEVELS = %w(debug info warn error fatal).freeze
 
   def self.simulate?
     %w(yes true 1).include? ENV.fetch('REX_SIMULATE', '').downcase
   end
 
+  def self.validate_settings!
+    super
+    self.validate_ssh_log_level!
+    @settings[:ssh_log_level] = @settings[:ssh_log_level].to_sym
+  end
+
+  def self.validate_ssh_log_level!
+    wanted_level = @settings[:ssh_log_level].to_s
+    unless SSH_LOG_LEVELS.include? wanted_level
+      raise "Wrong value '#{@settings[:ssh_log_level]}' for ssh_log_level, must be one of #{SSH_LOG_LEVELS.join(', ')}"
+    end
+
+    # regular log levels correspond to upcased ssh logger levels
+    ssh, regular = [wanted_level, SmartProxyDynflowCore::SETTINGS.log_level.to_s.downcase].map do |wanted|
```

Comment:
This is breaking on my foreman server side when trying to run the server with this code:

```
/home/inecas/Projects/ws/foreman-rex/foreman_remote_execution/lib/foreman_remote_execution_core.rb:35:in `validate_ssh_log_level!': uninitialized constant ForemanRemoteExecutionCore::SmartProxyDynflowCore (NameError)
	from /home/inecas/Projects/ws/foreman-rex/foreman_remote_execution/lib/foreman_remote_execution_core.rb:24:in `validate_settings!'
	from /home/inecas/Projects/ws/foreman-rex/foreman-tasks/lib/foreman_tasks_core/settings_loader.rb:41:in `initialize_settings'
	from /home/inecas/Projects/ws/foreman-rex/foreman-tasks/lib/foreman_tasks_core/settings_loader.rb:31:in `setup_settings'
	from /home/inecas/Projects/ws/foreman-rex/foreman-tasks/lib/foreman_tasks/engine.rb:128:in `block (2 levels) in <class:Engine>'
	from /home/inecas/Projects/ws/foreman-rex/foreman-tasks/lib/foreman_tasks/engine.rb:124:in `each_key'
	from /home/inecas/Projects/ws/foreman-rex/foreman-tasks/lib/foreman_tasks/engine.rb:124:in `block in <class:Engine>'

```